deploy/helm/secret-operator/values.yaml

```diff
@@ -3,6 +3,14 @@
 image:
   repository: docker.stackable.tech/stackable/secret-operator
   pullPolicy: IfNotPresent
+csiProvisionerImage:
+  repository: docker.stackable.tech/k8s/sig-storage/csi-provisioner
+  tag: v3.1.0
+  pullPolicy: IfNotPresent
+csiNodeDriverRegistrarImage:
```

Comment:
Very minor nitpick: I'd prefer to follow the scheme of `csiProvisioner.image.repository`, so that we have a clear opening for other sidecar-specific settings.